crates/storage/src/blueprint/merklized.rs

```diff
@@ -321,7 +338,7 @@ macro_rules! basic_merklelized_storage_tests {
             #[test]
             fn get() {
                 let mut storage = InMemoryStorage::default();
-                let mut structured_storage = StructuredStorage::new(&mut storage);
```

Comment:
These all still have the name `structured_storage` in all these tests. Is that still accurate?

Reply:
Oh. Hmm:
```
pub type StorageTransaction<S> = StructuredStorage<StorageTransactionInner<S>>;
```
Either way I guess...